Validators/DatiGeneraliDocumentoValidator.cs

```diff
@@ -22,7 +22,7 @@ public DatiGeneraliDocumentoValidator()
             RuleFor(x => x.Numero)
                 .Matches(@"\d")
                 .WithMessage("Numero non contiene caratteri numerici")
-                .WithErrorCode("00411");
+                .WithErrorCode("00425");
```

Comment:
idem c.s.

Reply:
L'ho trovato sia sul tracciato xml che sulle specifiche 1.3 pag 130 [https://www.agenziaentrate.gov.it/wps/file/Nsilib/Nsi/Schede/Comunicazioni/Fatture+e+corrispettivi/Fatture+e+corrispettivi+ST/ST+invio+di+fatturazione+elettronica/ST+Fatturazione+elettronica+-+Allegato+A+inglese/Appendix+A_Technical+Specifications+vers+1.3.pdf](url)